modules/nw-udn-examples.adoc

```diff
@@ -0,0 +1,69 @@
+//module included in the following assembly:
+//
+// *networking/multiple_networks/understanding-user-defined-networks.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nw-udn-examples_{context}"]
+= Examples of UserDefinedNetworks
```

Comment:
@ormergi PTAL at this new section I created to address your previous feedback about examples. If there are other details you believe the users needs please let me know.